app/src/main/java/com/ozonehis/eip/Application.java

```diff
@@ -7,17 +7,58 @@
  */
 package com.ozonehis.eip;
 
+import jakarta.annotation.PostConstruct;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import liquibase.database.Database;
+import liquibase.database.DatabaseFactory;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.lockservice.LockServiceFactory;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @Slf4j
 @SpringBootApplication(scanBasePackages = {"org.openmrs.eip, com.ozonehis.eip"})
 public class Application {
+    @Value("${spring.mngt-datasource.jdbcUrl:#{null}}")
+    private String url;
+
+    @Value("${spring.mngt-datasource.username:#{null}}")
+    private String username;
+
+    @Value("${spring.mngt-datasource.password:#{null}}")
+    private String password;
```

Comment:
There is no need to do this since we already have the management datasource class [here](https://github.com/openmrs/openmrs-eip/blob/master/commons/src/main/java/org/openmrs/eip/app/management/config/ManagementDataSourceConfig.java)